src/components/ec-navigation-arrows/ec-navigation-arrows.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    class="ec-navigation-arrows"
+    data-test="ec-navigation-arrows"
+  >
+    <button
+      type="button"
+      :class="leftButtonClasses"
+      :disabled="isLeftArrowDisabled"
+      class="tw-mr-12"
+      data-test="ec-navigation-arrows__left"
+      @click="emit(NavigationArrowsEvent.LEFT_ARROW_CLICK)"
+    >
+      <ec-icon :size="24" :name="IconName.SimpleChevronLeft" /></button>
+    <button
+      type="button"
+      :class="rightButtonClasses"
+      :disabled="isRightArrowDisabled"
+      data-test="ec-navigation-arrows__right"
+      @click="emit(NavigationArrowsEvent.RIGHT_ARROW_CLICK)"
+    >
+      <ec-icon :size="24" :name="IconName.SimpleChevronRight" /></button>
+  </div>
+</template>
+
+<script setup lang="ts">
+
+import { computed } from 'vue';
+
+import EcIcon from '../ec-icon/ec-icon.vue';
+import { IconName } from '../ec-icon/types';
+import { NavigationArrowsEvent } from './types';
+
+const emit = defineEmits<{
+  'right-arrow-click': []
+  'left-arrow-click': [],
```

Comment:
If the component is called navigation arrows then the events could also be called `previous` and `next`.